model_analyzer/triton/server/server_docker.py

```diff
@@ -82,10 +85,15 @@ def start(self):
         server_grpc_port = self._server_config['grpc-port'] or 8001
         server_metrics_port = self._server_config['metrics-port'] or 8002
 
+        # Docker ports
+        docker_http_port = self._docker_ports['http_port'] or 8000
+        docker_grpc_port = self._docker_ports['grpc_port'] or 8001
+        docker_metrics_port = self._docker_ports['metrics_port'] or 8002
+
         ports = {
-            server_http_port: LOCAL_HTTP_PORT,
-            server_grpc_port: LOCAL_GRPC_PORT,
-            server_metrics_port: LOCAL_METRICS_PORT
+            server_http_port: docker_http_port,
```

Comment:
Because we set both to the ports coming in from args.xxx_endpoint, we might as well just tell triton to use the free ports.

Reply:
Because I didn't change the `TritonConfig` in the caller function `server_grpc_port` will be equal to 8000 always. I think that is Ok because we have the full control of the applications inside the container. I can set the ports inside the container to `8000,8001,8002` always so that the code is more readable.